app/views/articles/show.html.erb

```diff
@@ -1,15 +1,15 @@
 <div class="article-show">
-<p>
-    <strong>Title:</strong>
-    <%= @article.title %>
-</p>
-<p>
-    <strong></strong>
+    <div class="title-show">
+      <%= @article.title %>
+    </div>
+      <div class="lead">
```

Comment:
indentation is off

Reply:
done

Reply:
the indentation is still off in line 5-6-7

Reply:
done